packages/app-desktop/gui/styles/index.scss

```
@@ -2,6 +2,7 @@
 @use './base-text.scss';
 @use './base-button.scss';
 @use './dialog-modal-layer.scss';
+@use './editor-mode-selector-dialog.scss';
```

Stylelint (17.3.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)